docs/reference/src/language/builtins/elements.md

```diff
@@ -656,6 +656,78 @@ export component Example inherits Window {
 }
 ```
 
+## `SwipeGestureRecognizer`
+
+The `SwipeGestureRecognizer` is used to recognize swipes gesture on the screen in some particular direction.
+Press events on it are still forwarded to the children, but with a small delay.
+If the cursor has moved by more than some internal threshold, a swipe gesture is recognized, and events are no
+longer forwared to the children.
+
+### Properties
+
+-   **`enabled`** (_in_ _bool_): When not enabled, the `SwipeGestureRecognizer` is not recognizing any events.
+    (default value: `true`)
+-   **`swipe-left`**, **`swipe-right`**, **`swipe-up`**, **`swipe-down`** (_out_ _bool_): Recognizes a swipe in
+    the corresponding direction. (default value: `false`)
+-   **`pressed-position`** (_out_ _Point_): The position of the cursor when the swipe started.
+-   **`current-position`** (_out_ _Point_): The current position of the cursor.
+-   **`swiping`** (_out_ _bool_): `true` when the cursor is pressed and the threshold was reached. (default value: `false`)
+
+### Callbacks
+
+-   **`moved()`**: Called when the cursor is moved.
+-   **`swiped()`**: Called when the cursor is released and the swipe is finished.
```

Comment:
```suggestion
-   **`swiped()`**: Invoked after the swipe gesture was recognised and the pointer was released.
```

I'm a little bit confused here though. The docs say that this is invoked when the cursor is released, but the example uses the `pressed-position` in the callback handler.

I see the different options:

- Document that `pressed-position` in fact holds the _last_ pressed position, even after lifting the pointer?
- Add named parameters or an enum to this callback that indicates the direction of the recognised gesture to simplify the typical handler that just wants to know which direction was swiped?

Reply:
yes indeed, pressed-position is still valid at this time and stays like so until the pointer is pressed again.

Reply:
Okay, makes sense. The docs refer to the start of the gestures, so that's all good then.